### DISCUSSION TITLE
Backend database - Instead of EF is it opssible to use Dapper?

### DISCUSSION THREAD

**Discussion by StargazerNC** [Azure/azure-mobile-apps#481] [Q&A] [answered]
Hello!

It seems to me that this may be possible to do, but I'm failing to find resources on the "how".

So, instead of using Entity Framework with Azure SQL Server, I would like to use for example Dapper on the backend. Is there any documentation on how to achieve this?

Thanks!

**Comment by adrianhall** [comment, marked as answer]
No direct documentation on how to do this.  However, you need to implement a repository (for which there is documentation - see my blog at https://adrianhall.github.io)

Your major challenge, however, will be getting a LINQ-compatible interface to the data table for the list operation.  There are things like https://github.com/ryanwatson/Dapper.Extensions.Linq that might help.

**Comment by StargazerNC** [comment]
HI @adrianhall 
Thank you for your reply. I've glanced over the content you suggested and it seems and can use for my objectives. Thank you again.
